src/controllers/update/update.manage.trusts.tell.us.about.the.former.bo.controller.ts

```diff
@@ -50,8 +51,12 @@ export const post = async (req: Request, res: Response, next: NextFunction) => {
 
     // check for form validation errors
     const errorList = validationResult(req);
-    if (!errorList.isEmpty()) {
-      const pageProperties = getPageProperties(trust, formData, formatValidationError(errorList.array()));
+    const errors = await checkErrors(req);
```

Comment:
This variable could be renamed to `filingPeriodDateErrors`.

Reply:
This name is the same in all controllers because the job is similar: to get validation errors that are not in errorList.
The name is as common as can be errorList = validationResult(req)

Reply:
I like variable names to be as specific as possible, but up to you.